content/docs/context.md

```diff
@@ -59,6 +59,10 @@ A React component that allows Consumers to subscribe to context changes.
 
 Accepts a `value` prop to be passed to Consumers that are descendants of this Provider. One Provider can be connected to many Consumers. Providers can be nested to override values deeper within the tree.
 
+> Note:
+>
+> If `value` is not provided, Consumers receive `undefined`. To use the default value in a Consumer instead, remove the Provider altogether.
+
```

Comment:
I feel like this note is more confusing than what it explains. It seems to *suggest* removing a Provider as a solution to some problem but usually you don't want to do that. Maybe there's a way to reword where it doesn't sound like we *recommend* removing Providers.